src/core/Akka.Cluster/ClusterDaemon.cs

```diff
@@ -924,13 +917,13 @@ private void CreateChildren(Cluster cluster)
     ///
     /// Actor used to power the guts of the Akka.Cluster membership and gossip protocols.
     /// </summary>
-    internal class ClusterCoreDaemon : UntypedActor, IRequiresMessageQueue<IUnboundedMessageQueueSemantics>
+    internal sealed class ClusterCoreDaemon : UntypedActor, IRequiresMessageQueue<IUnboundedMessageQueueSemantics>, IWithTimers
```

Comment:
Always try to seal classes that shouldn't be inherited. Also, added `IWIthTimers` support here as part of resolving AK1004.